webserver/testing.py

```diff
@@ -1,8 +1,14 @@
 from db.testing import DatabaseTestCase
 
+from brainzutils.ratelimit import set_rate_limits
 
 class ServerTestCase(DatabaseTestCase):
 
+
+    def setUp(self):
+        super(ServerTestCase, self).setUp()
+        set_rate_limits(1000, 1000, 10000)
```

Comment:
instead of selecting large magic numbers here, it'd be nice to be able to actually disable the rate-limiting during tests. Can we add a ticket to BU to add this?